openAPI/core-back-internal.yaml

```diff
@@ -271,7 +271,24 @@ paths:
         uri:
           Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${CheckExistingIdentityFunction.Arn}:live/invocations
         passthroughBehavior: "when_no_match"
-        type: "aws_proxy"
+        type: "aws"
+        requestTemplates:
+          application/x-www-form-urlencoded:
```

Comment:
Wouldn't this be `application/json` ?

Reply:
Apparently not. When I checked it's called with application/x-www-form-urlencoded.